alibi/confidence/model_linearity.py

```diff
@@ -341,10 +349,16 @@ def _infer_feature_range(X_train: np.ndarray) -> np.ndarray:
     return np.vstack((X_train.min(axis=0), X_train.max(axis=0))).T
 
 
-class LinearityMeasure(object):
+class LinearityMeasure:
```

Comment:
Removed the old-style ineheritance from `object` as it's not necessary in Python 3.